src/libServer/JSONConversion.cpp

```diff
@@ -39,46 +39,17 @@ using namespace boost::multiprecision;
 
 unsigned int JSON_TRAN_OBJECT_SIZE = 10;
 
-const Json::Value JSONConversion::convertBoolArraytoJson(
```

Comment:
It's not used however. And the usage is very limited, only applicable when we need a vector of bool, which is used for the previous isMicroBlockEmpty only, remove in case of using wrongly.

Reply:
I dont see anyway it being used wrongly. Can be a useful piece of code for reference.

Reply:
It's better to put util function into libUtils. As other functions in this class are all very specific.